resource_virtual_server.go

```diff
@@ -322,6 +329,7 @@ func resourceVirtualServerSet(d *schema.ResourceData, meta interface{}) error {
 	setBool(&r.SSL.AddHTTPHeaders, d, "ssl_add_http_headers")
 	setBool(&r.Basic.SSLDecrypt, d, "ssl_decrypt")
 	setString(&r.SSL.ServerCertDefault, d, "ssl_server_cert_default")
+        setString(&r.SSL.ServerCertAltCertificates, d, "ssl_server_cert_alt_certificates")
```

Comment:
Good catch - it should be setStringList, not setString, because the API expects a list of strings instead of a bare string. I'll push a commit to fix that.